packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -708,17 +708,28 @@ export class GarbageCollector implements IGarbageCollector {
 			return;
 		}
 
+		// If sweep is disabled, we'll tombstone both tombstone-ready and sweep-ready nodes.
```

Comment:
Maybe add that this is important because if a container loads after sweep timeout, those nodes would directly become sweep-ready skipping over tombstone-ready. Also, with the plan to always make nodes go through tombstones first, this won't be needed right.

Reply:
Yeah I guess you're right, we could theoretically add an assert at that point saying that all SweepReady nodes are in the Tombstone list.  But maybe in practice there are edge cases or something.

I'll update the comment.